common/common-ui/src/main/res/drawable/ic_paste.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+  ~ Copyright (c) 2023 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This drawable was added to the common module, but the small icon for the QR was added to the sync module. 
@CrisBarreiro is `ic_paste` used anywhere else? if not, then I’d move it to the sync module instead.

Reply:
It’s not but I added it here as a counterpart to `ic_copy` which is used in other places